### DISCUSSION TITLE
I accidentally added an access token to my GitHub public repository

### DISCUSSION THREAD

**Discussion by edith-mark** [community/community#21685] [New to GitHub]
<p>Hi,</p>
<p>I accidentally added an access token to my GitHub public repo.<br>
Five minutes later I saw that and I reverted that commit. also, I deleted that repository and access token. <br>
Do I need to worry about now? Is there anything I should definitely know about this issue?<br>
This all happened 18 days ago. is there any chance of that token still exist in GitHub after deleting that repository.</p>
<p>Thanks</p>

**Comment by Gisgar3** [comment, marked as answer]
<p>That access-token should no longer exist anywhere on GitHub, as the commits were reverted and the repository was deleted. However, I highly recommend that you  <strong>refresh</strong> your access-token to make sure that you have a new token to use incase the old one was taken at some point.</p>
<p>Hope this helps!</p>
